RecoEcal/EgammaClusterAlgos/src/PFECALSuperClusterAlgo.cc

```diff
@@ -251,24 +268,69 @@ void PFECALSuperClusterAlgo::run() {
   buildAllSuperClusters(_clustersEE, threshPFClusterSeedEndcap_);
 }
 
-void PFECALSuperClusterAlgo::buildAllSuperClusters(CalibClusterPtrVector& clusters, double seedthresh) {
+void PFECALSuperClusterAlgo::buildAllSuperClusters(CalibratedClusterPtrVector& clusters, double seedthresh) {
+  if (_clustype == PFECALSuperClusterAlgo::kMustache)
```

Comment:
In the previous code this was also for  `_clustype == PFECALSuperClusterAlgo::kBOX`: isn't that option used any more? If so, better to clean-up rather than letting it idle (ok, in a different PR)

Reply:
you are right, the kBOX option is still present in other place and logically it follows the building of the Mustache. 
The only change is in the `isClustered` code. 

I'm going to fix this now for consistency.. and I will change also the code for the other comments :)